fastd-exporter.go

```diff
@@ -342,12 +353,24 @@ func main() {
 	}
 
 	for i := 0; i < len(instances); i++ {
-		config, err := parseConfig(instances[i])
+		instance := strings.SplitN(instances[i], "=", 2)
```

Comment:
Regex for input validation?

Reply:
i've changed it up and there is now input validation. Hoewer, i didn't cover everything that would've worked previously.